contracts/staking/test/utils/Simulation.ts

```diff
@@ -1,9 +1,11 @@
+/*
```

Comment:
Will this be updated in this PR, or another one?

Reply:
The simulations will be the last to get updated, after the implementation is finalized and the unit tests are done. Likely, next week.